mirascope/core/bedrock/__init__.py

```diff
@@ -0,0 +1,24 @@
+from ._call import bedrock_call
+from ._call import bedrock_call as call
+from ._utils import AssistantMessageTypeDef, BedrockMessageParam, UserMessageTypeDef
+from .call_params import BedrockCallParams
+from .call_response import BedrockCallResponse
+from .call_response_chunk import BedrockCallResponseChunk
+from .dynamic_config import BedrockDynamicConfig
+from .stream import BedrockStream
+from .tool import BedrockTool, BedrockToolConfig
+
```

Comment:
missing the message param type

for reference, see [this](https://github.com/Mirascope/mirascope/blob/e357897c40d2e4923a8f022bac79c878d65b8811/mirascope/core/openai/__init__.py#L17)